src/scripts/OSFramework/OSUI/Helper/Dom.ts

```diff
@@ -437,6 +437,22 @@ namespace OSFramework.OSUI.Helper {
 			return [..._filteredElements] as HTMLElement[];
 		}
 
+		/**
+		 * Method to check if element is inside a Popup widget
+		 *
+		 * @static
+		 * @param {HTMLElement} element
+		 * @return {*}  {boolean}
+		 * @memberof Dom
+		 */
+		public static IsInsidePopupWidget(element: HTMLElement): boolean {
+			const _popup = document.querySelector(Constants.Dot + GlobalEnum.CssClassElements.Popup);
```

Comment:
Can't we've more than one popup? If so, we should look by the popup that's open/active, right?